rosdep/base.yaml

```
@@ -2982,6 +2982,12 @@ libqwt6:
   fedora: [qwt-devel]
   gentoo: ['x11-libs/qwt:6']
   ubuntu: [libqwt-dev]
+libqwt6-qt5-dev:
```

```suggestion
libqwt-qt5-dev:
```

Our default is to follow the debian name for the key so we should do that here too.